### DISCUSSION TITLE
Keycloak version 15 - kc.sh ?

### DISCUSSION THREAD

**Discussion by mchingotto** [sventorben/keycloak-home-idp-discovery#208] [Q&A]
Hello,

   Sorry for asking, I am running out of time.
   I would like to install this lib but I can not find rc.sh anywhere in the keycloak installation. Why ? I saw upgrades procedures from 15 where exists the execution of kc.sh.

We are running in the test bed 15.0.1, for sure We should upgrade soon.
Any input appreciated.

On the other hand, Is it a crazy idea to use this lib with version 16 until We upgrade to a newer KC ?

Thank you,
M

**Comment by sventorben** [comment]
Hello @mchingotto 

kc.sh script only exist in Quarkus-based distributions (>= 15.1). Simply skip that step if you are on version 15.0. It should be sufficient to copy the jar file to the correct location.

I think version 16 should be backwards compatible with Keycloak version 15. Simply give it a try.

Regards
Sven-Torben

**Comment by mchingotto** [comment]
Hello @sventorben !!

Thank you !! I appreciate your answer so fast.
I will let it a try and I will give you feedback about that.

Thank you for the time you dedicated on this.
Warm regards,
Mariano

**Comment by mchingotto** [comment]
Up and Running with JVM 11, testing now.
Let me back to you ASAP.
